infrastructure/cdn-in-a-box/traffic_ops/run-go.sh

```diff
@@ -80,7 +80,8 @@ CDNCONF=/opt/traffic_ops/app/conf/cdn.conf
 DBCONF=/opt/traffic_ops/app/conf/production/database.conf
 RIAKCONF=/opt/traffic_ops/app/conf/production/riak.conf
 mkdir -p /var/log/traffic_ops
-touch /var/log/traffic_ops/traffic_ops.log
+touch "$TO_LOG_ERROR" "$TO_LOG_WARNING" "$TO_LOG_INFO" "$TO_LOG_DEBUG" "$TO_LOG_EVENT"
+tail -qf "$TO_LOG_ERROR" "$TO_LOG_WARNING" "$TO_LOG_INFO" "$TO_LOG_DEBUG" "$TO_LOG_EVENT" &
```

Comment:
I almost always want to start a bash script with [Unofficial Strict Mode](http://redsymbol.net/articles/unofficial-bash-strict-mode/). But specifically here, the _nounset_ flag:

`set -o nounset` (or `set -u` but I always prefer the longer, descriptive flags in scripts)

It will help ensure the variables we are relying on are set, and will fail if that isn't the case. Definitely helped me in the past. `pipefail` and `errexit` are also critical I feel, but I'm not sure what all paths we expect from this script.

Reply:
Agreed on all points.
* Adjusted various scripts so we can use `nounset` when `errexit` is set in [`fd8854ace5~...906f79b27f`](/apache/trafficcontrol/compare/fd8854ace5~...906f79b27f)
* Setting `errexit`, `nounset`, and `pipefail` in 48ab9744ed